docs/presentation/CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+## Contributing
```

Comment:
Why this file, is it relevant for others?

Reply:
It's part of Reveal.js which @Qqwy used to make the presentation slides. The presentation files aren't really related to the rest of the project but I'm not sure which can be safely removed